packages/ember-glimmer/tests/integration/custom-component-manager-test.js

```diff
@@ -0,0 +1,27 @@
+import { moduleFor, RenderingTest } from '../utils/test-case';
+import TestComponentManager from './helpers/test-component-manager';
+import {
+  EMBER_GLIMMER_ALLOW_BACKTRACKING_RERENDER,
+  GLIMMER_CUSTOM_COMPONENT_MANAGER,
+  MANDATORY_SETTER
+} from 'ember/features';
+
+if (GLIMMER_CUSTOM_COMPONENT_MANAGER) {
+  moduleFor('Components test: curly components with custom manager', class extends RenderingTest {
+    ['@test it can render a basic component with custom component manager']() {
+      this.owner.register('component-manager:glimmer', new TestComponentManager());
+      this.registerComponent('foo-bar', {
+        template: '{{use-component-manager "glimmer"}}hello',
+        managerId: 'glimmer'
```

Comment:
We probably shouldn't use `glimmer` here, because we actually intend to use that managerId in the future and it may confuse future readers of the test about what we are talking...